README.md

```diff
@@ -63,3 +63,45 @@ Any help you can provide is greatly appreciated!
 If you have problems, suggestions or other feedback you can open an [issue](https://github.com/mathiasvr/bluejay/issues).
 
 Join the new [Discord server](https://discord.gg/phAmtxnMMN) to ask questions and discuss Bluejay.
+
+### IDE
+To install an IDE and toolchain, use [SILABS Simplicity Studio](https://www.silabs.com/developers/simplicity-studio).
+
+After installing and starting Simplicity, choose *"Install"* from the toolbar, then *"Manage installed packages"* from there select the *"SDK"* tab and install the 8051 SDK. From the *"Toolchains"* tab install the *"Keil 8051 Toolchain"*.
+
+#### Obtaining license key
+You now only have the evaluation license for the Keil toolchain, you need to get a license for the full version. Go to "Help" -> "Licensing" -> "Keil 8051". From there you need to follow the link to a from after which you will be sent a license key for the full version of the Keil toolchain.
+
+### Registering via WINE
+If you only want compiler and linker, [download the C51 package from Keil](https://www.keil.com/demo/eval/c51.htm). Run it with wine:
```

Comment:
Is this for free like the Silabs one? Otherwise, can we install simplicity studio like this instead of uVision.

Reply:
Yes, this is free. The problem is, that with simplicity it seems to use its own instance of wine, so I can not use the toolchain for docker - I guess it has its own registry... At least when I try to run the tools with my local wine, I still get the error that I don't have the licence although it is added in simplicity. Maybe I am missing something?

Reply:
Hm, I think it should work when setting the wineprefix to the simplicity wine folder, but anyway, if this is free it's fine as well.

Reply:
@stylesuxx with simplicity you need to specify WINE_PREFIX

![image](https://user-images.githubusercontent.com/265528/108644115-a6c85380-74d3-11eb-90ed-ccac17b3aaa9.png)

A docker environment with password protected Simplicty config for the purposes of CI/CD can be useful I think